http-lambda-invoker.go

```diff
@@ -15,17 +15,20 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"strings"
 )
 
 // LambdaClient enables mocking of the client for test purposes
 type LambdaClient struct {
 	lambdaiface.LambdaAPI
 }
 
+type proxyHeader map[string]string
+
 // Parts of the request to send to Lambda.
 type makeProxyRequest struct {
 	Body              []byte              `json:"body"`
-	Headers           map[string][]string `json:"headers"`
+	Headers           proxyHeader					`json:"headers"`
```

Comment:
If you're using VSCode, please try https://github.com/microsoft/vscode-go for automatic formatting.

Reply:
I'm not exactly sure what it is you want me to change here

Reply:
Oh sorry, it's the formatting. The mentioned plugin will take care of that automatically.